sentry-anyhow/src/lib.rs

```diff
@@ -4,8 +4,11 @@
 //!
 //! ```no_run
 //! # fn function_that_might_fail() -> anyhow::Result<()> { Ok(()) }
-//! use sentry_anyhow::capture_anyhow;
+//! use sentry_anyhow::{capture_anyhow, AnyhowIntegration};
```

Comment:
_nit_: Consider changing this example to include the function and have it return an error. Then you can simply capture the error and report it, no more return:


```rust
fn function_that_might_fail() -> anyhow::Result<()> { 
    Err("this is an error".into()) 
}

let _sentry = sentry::init(sentry::ClientOptions::new().add_integration(AnyhowIntegration));
if let Err(error) = match function_that_might_fail() {
    capture_anyhow(&err);
}
```